camel/utils/token_counting.py

```diff
@@ -21,6 +21,7 @@
 from typing import TYPE_CHECKING, List, Optional
 
 from anthropic import Anthropic
+from litellm import completion_cost, token_counter
```

Comment:
move this part into `class LiteLLMTokenCounter` so it will only be imported when necessary

Reply:
👌